File: akka-scala/src/main/scala/io/swagger/client/api/PostItServiceApi.scala
package io.swagger.client.api

import io.swagger.client.model.PostitCreatePostItTableResponse
import io.swagger.client.model.PostitCreatePostItTable
import io.swagger.client.model.PostitDeleteApiKey
import io.swagger.client.model.PostitDeleteApiKeyResponse
import io.swagger.client.model.PostitPostItResponse
import io.swagger.client.model.PostitPostIt
import io.swagger.client.model.PostitRegenerateApiKeyResponse
import io.swagger.client.model.PostitRegenerateApiKey
import io.swagger.client.model.PostitUpdatePostItTable
import io.swagger.client.model.PostitUpdatePostItTableResponse
import io.swagger.client.core._
import io.swagger.client.core.CollectionFormats._
import io.swagger.client.core.ApiKeyLocations._

object PostItServiceApi {

  /**
   * 
   * 
   * Expected answers:
   *   code 0 : PostitCreatePostItTableResponse (Description)
   * 
   * @param body 
   */
  def createPostItTable(body: PostitCreatePostItTable): ApiRequest[PostitCreatePostItTableResponse] =
    ApiRequest[PostitCreatePostItTableResponse](ApiMethods.POST, "http://localhost", "/api/1.0/create", "application/json")
      .withBody(body)
      .withDefaultSuccessResponse[PostitCreatePostItTableResponse]
      
  /**
   * 
   * 
   * Expected answers:
   *   code 0 : PostitDeleteApiKeyResponse (Description)
   * 
   * @param body 
   */
  def deleteApiKey(body: PostitDeleteApiKey): ApiRequest[PostitDeleteApiKeyResponse] =
    ApiRequest[PostitDeleteApiKeyResponse](ApiMethods.POST, "http://localhost", "/api/1.0/delete", "application/json")
      .withBody(body)
      .withDefaultSuccessResponse[PostitDeleteApiKeyResponse]
      
  /**
   * 
   * 
   * Expected answers:
   *   code 0 : PostitPostItResponse (Description)
   * 
   * @param body 
   */
  def postIt(body: PostitPostIt): ApiRequest[PostitPostItResponse] =
    ApiRequest[PostitPostItResponse](ApiMethods.POST, "http://localhost", "/api/1.0/post", "application/json")
      .withBody(body)
      .withDefaultSuccessResponse[PostitPostItResponse]
      
  /**
   * 
   * 
   * Expected answers:
   *   code 0 : PostitRegenerateApiKeyResponse (Description)
   * 
   * @param body 
   */
  def regenerateApiKey(body: PostitRegenerateApiKey): ApiRequest[PostitRegenerateApiKeyResponse] =
    ApiRequest[PostitRegenerateApiKeyResponse](ApiMethods.POST, "http://localhost", "/api/1.0/regenerate", "application/json")
      .withBody(body)
      .withDefaultSuccessResponse[PostitRegenerateApiKeyResponse]
      
  /**
   * 
   * 
   * Expected answers:
   *   code 0 : PostitUpdatePostItTableResponse (Description)
   * 
   * @param body 
   */
  def updatePostItTable(body: PostitUpdatePostItTable): ApiRequest[PostitUpdatePostItTableResponse] =
    ApiRequest[PostitUpdatePostItTableResponse](ApiMethods.POST, "http://localhost", "/api/1.0/update", "application/json")
      .withBody(body)
      .withDefaultSuccessResponse[PostitUpdatePostItTableResponse]
      


}

File: aspnet5/src/IO.Swagger/Controllers/PostItServiceApi.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel;
using System.IO;
using System.Linq;
using System.Net;
using System.Threading.Tasks;
using Microsoft.AspNet.Mvc;
using Newtonsoft.Json;
using Swashbuckle.SwaggerGen.Annotations;
using IO.Swagger.Models;

namespace IO.Swagger.Controllers
{ 
    /// <summary>
    /// 
    /// </summary>
    public class PostItServiceApiController : Controller
    { 

        /// <summary>
        /// PostItService.createPostItTable
        /// </summary>
        
        /// <param name="body"></param>
        /// <response code="0">Description</response>
        [HttpPost]
        [Route("/api/1.0/create")]
        [SwaggerOperation("CreatePostItTable")]
        [SwaggerResponse(200, type: typeof(PostitCreatePostItTableResponse))]
        public IActionResult CreatePostItTable([FromBody]PostitCreatePostItTable body)
        { 
            string exampleJson = null;
            
            var example = exampleJson != null
            ? JsonConvert.DeserializeObject<PostitCreatePostItTableResponse>(exampleJson)
            : default(PostitCreatePostItTableResponse);
            
            return new ObjectResult(example);
        }


        /// <summary>
        /// PostItService.deleteApiKey
        /// </summary>
        
        /// <param name="body"></param>
        /// <response code="0">Description</response>
        [HttpPost]
        [Route("/api/1.0/delete")]
        [SwaggerOperation("DeleteApiKey")]
        [SwaggerResponse(200, type: typeof(PostitDeleteApiKeyResponse))]
        public IActionResult DeleteApiKey([FromBody]PostitDeleteApiKey body)
        { 
            string exampleJson = null;
            
            var example = exampleJson != null
            ? JsonConvert.DeserializeObject<PostitDeleteApiKeyResponse>(exampleJson)
            : default(PostitDeleteApiKeyResponse);
            
            return new ObjectResult(example);
        }


        /// <summary>
        /// PostItService.postIt
        /// </summary>
        
        /// <param name="body"></param>
        /// <response code="0">Description</response>
        [HttpPost]
        [Route("/api/1.0/post")]
        [SwaggerOperation("PostIt")]
        [SwaggerResponse(200, type: typeof(PostitPostItResponse))]
        public IActionResult PostIt([FromBody]PostitPostIt body)
        { 
            string exampleJson = null;
            
            var example = exampleJson != null
            ? JsonConvert.DeserializeObject<PostitPostItResponse>(exampleJson)
            : default(PostitPostItResponse);
            
            return new ObjectResult(example);
        }


        /// <summary>
        /// PostItService.regenerateApiKey
        /// </summary>
        
        /// <param name="body"></param>
        /// <response code="0">Description</response>
        [HttpPost]
        [Route("/api/1.0/regenerate")]
        [SwaggerOperation("RegenerateApiKey")]
        [SwaggerResponse(200, type: typeof(PostitRegenerateApiKeyResponse))]
        public IActionResult RegenerateApiKey([FromBody]PostitRegenerateApiKey body)
        { 
            string exampleJson = null;
            
            var example = exampleJson != null
            ? JsonConvert.DeserializeObject<PostitRegenerateApiKeyResponse>(exampleJson)
            : default(PostitRegenerateApiKeyResponse);
            
            return new ObjectResult(example);
        }


        /// <summary>
        /// PostItService.updatePostItTable
        /// </summary>
        
        /// <param name="body"></param>
        /// <response code="0">Description</response>
        [HttpPost]
        [Route("/api/1.0/update")]
        [SwaggerOperation("UpdatePostItTable")]
        [SwaggerResponse(200, type: typeof(PostitUpdatePostItTableResponse))]
        public IActionResult UpdatePostItTable([FromBody]PostitUpdatePostItTable body)
        { 
            string exampleJson = null;
            
            var example = exampleJson != null
            ? JsonConvert.DeserializeObject<PostitUpdatePostItTableResponse>(exampleJson)
            : default(PostitUpdatePostItTableResponse);
            
            return new ObjectResult(example);
        }
    }
}
